crates/subspace-farmer/src/single_disk_farm.rs

```diff
@@ -1198,18 +1198,27 @@ impl SingleDiskFarm {
 
         {
             let plot = directory.join(Self::PLOT_FILE);
+            if !plot.exists() {
+                return Ok(());
+            }
```

Comment:
I think it should still try to delete other files if this one doesn't exist, not returning success

Reply:
I see, fixing it in a min.